Modules/System/Extension Management/src/ExtensionSettings.Page.al

```diff
@@ -22,6 +22,7 @@ page 2511 "Extension Settings"
         {
             group(Group)
             {
+                ShowCaption = false;
```

Comment:
Seems like a caption was added recently. 
`Caption = 'General';`

I'll go ahead and process the rest of the PR. If you could just remove this and I'll merge the PR in.

Reply:
I wasn't sure what you expect me to do.

Remove `ShowCaption = false;` or what do you mean and we keep the "Caption=General", that isn't released yet, but part of the internal master?